app/src/main/java/com/fivegmag/a5gmscommonlibrary/models/ServiceAccessInformation.kt

```diff
@@ -16,10 +16,20 @@ import kotlinx.parcelize.Parcelize
 data class ServiceAccessInformation(
     val provisioningSessionId : String,
     val provisioningSessionType: String?,
-    val streamingAccess: StreamingAccess
+    val streamingAccess: StreamingAccess,
+    var clientConsumptionReportingConfiguration: ClientConsumptionReportingConfiguration
 ) : Parcelable
 
 @Parcelize
 data class StreamingAccess(
     val entryPoints: ArrayList<EntryPoint>?
 ) : Parcelable
+
+@Parcelize
+data class ClientConsumptionReportingConfiguration(
+    val serverAddresses : Array<String>,
+    val locationReporting: Boolean,
+    val samplePercentage: UInt,
```

Comment:
This is a float from 0.0 to 100.0

Reply:
Thanks @dsilhavy for the review and the comments. I revised the type accordingly.
And thanks @rjb1000 for helping changing the title.